oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/ElasticSearch.java

```diff
@@ -68,4 +68,18 @@ enum AnalyzerType {
             }
         }
     }
+
+    @Target({ElementType.FIELD})
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface Column {
+
+        /**
+         * Warning: this is only used to solve the excited column conflict since we need support merge all metrics
+         * in one index or sharding storage type both. If create a new a column, we should avoid the compatibility issue
+         * between these 2 storage type rather than use this alias.
```

Comment:
```suggestion
         * Warning: this is only used to solve the conflict among the exciting columns since we need support to merge all metrics
         * in one physical index template. When create a new column, we should avoid the compatibility issue
         * between these 2 storage modes rather than use this alias.
```